### PR TITLE
HKG: DBC for Hyundai Palisade and Kia Telluride 2023-24 (HDA2)

### PR DESCRIPTION
**Pull request by sunnyhaibin**
**Prerequisite of**
- https://github.com/commaai/openpilot/pull/27392
- https://github.com/commaai/openpilot/pull/27478

### CODE REVIEW

**Review by adeebshihadeh — CHANGES_REQUESTED** [review]
Can you factor out all the shared stuff with the generator?

File: hyundai_palisade_2023.dbc [hunk 1]
```diff
@@ -0,0 +1,1088 @@
...
+ SG_ LKA_ICON : 38|2@1+ (1,0) [0|0] "" XXX
+ SG_ LKA_MODE : 24|3@1+ (1,0) [0|0] "" XXX
+ SG_ LKA_WARNING : 32|1@1+ (1,0) [0|0] "" XXX
+ SG_ NEW_SIGNAL_1 : 27|2@1+ (1,0) [0|0] "" XXX
+ SG_ NEW_SIGNAL_2 : 70|2@0+ (1,0) [0|0] "" XXX
+ SG_ NEW_SIGNAL_3 : 111|8@0+ (1,0) [0|0] "" XXX
+ SG_ SET_ME_1 : 80|1@0+ (1,0) [0|0] "" XXX
+ SG_ STEER_MODE : 65|3@1+ (1,0) [0|0] "" XXX
+ SG_ STEER_REQ : 52|1@1+ (1,0) [0|0] "" XXX
+ SG_ TORQUE_REQUEST : 41|11@1+ (1,-1024) [0|0] "" XXX
```
Comment (sshane) [inline hyundai_palisade_2023.dbc:79]: This looks identical to LKAS in hyundai_canfd, let's make a base CAN FD DBC and put this in there.

File: hyundai_palisade_2023.dbc [hunk 2]
```diff
@@ -0,0 +1,1088 @@
...
+ SG_ CF_Datc_PwrInf : 32|4@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearAutoDisp : 40|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearBlwDisp : 52|4@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearChgReqDisp : 46|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearClimateScnDisp : 44|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearManual : 38|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearModDisp : 48|4@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearOffDisp : 42|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_TempDispUnit : 0|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Mtc_MaxAcDisp : 28|2@1+ (1,0) [0|0] "" XXX
```
Comment (sshane) [inline hyundai_palisade_2023.dbc:62]: Is this also a duplicate on the one in hyundai_kia_generic?

Comment (sunnyhaibin) [reply hyundai_palisade_2023.dbc:62]: Yes

Comment (sunnyhaibin) [reply hyundai_palisade_2023.dbc]: @sshane The platform has this message, but I'm not sure how relevant the definitions are or the use cases that would benefit us. We can remove this for the time being.

Comment (sshane) [reply hyundai_palisade_2023.dbc]: If the definition looks wrong, let's keep it in hyundai_kia_generic

**Comment by sunnyhaibin** [comment]
@sshane Is this what we want? Wanted to double-check before I sync with the latest.

**Pull request closed without merging**